Content.Server/Doors/Systems/AirlockSystem.cs

```diff
@@ -149,8 +149,8 @@ private void OnBeforeDoorDenied(EntityUid uid, AirlockComponent component, Befor
 
         private void OnActivate(EntityUid uid, AirlockComponent component, ActivateInWorldEvent args)
         {
-            if (TryComp<WiresPanelComponent>(uid, out var panel) && panel.Open &&
-                TryComp<ActorComponent>(args.User, out var actor))
+            if (TryComp<WiresPanelComponent>(uid, out var panel) && panel.Open && panel.WiresAccessible
```

Comment:
This only protects the wires if the player presses Use on the door with an open panel

Directly using the Wirecutter or Multitool on the door bypasses this and the plates do nothing!

Reply:
Good catch!